lib/omnibus/software.rb

```diff
@@ -696,6 +750,11 @@ def overridden?
       @overrides.key?(:version) && (@overrides[:version] != @version)
     end
 
+    def build_dep
+      @build_dep
+    end
+    expose :build_dep
```

Comment:
I couldn't name it that because there already exists a build_dependency method that is exposed and does something different. I'm totally down with finding a better way of doing that, but couldn't figure it out... such metaprogramming...